deployment/HDI-Spark/Notebooks/install-interactive-notebook.sh

```diff
@@ -11,6 +11,10 @@ set -e
 if  [[ $HOSTNAME == hn* ]] ;
 then
     # Update Livy Jars
+    sudo rm -rf /usr/hdp/current/livy2-server/jars/*
```

Comment:
can we back these up instead of completely removing them?

Reply:
Sure, I have changed the command to back these jars up in folder `old`.